src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2002,6 +2002,8 @@ internal enum ErrorCode
 
         #endregion
 
+        ERR_NoAwaitOnAsyncEnumerable = 8975
```

Comment:
This error code is now used. You'll need to merge latest `main` and update it. Also, keep the trailing comma